.coveragerc_appveyor

```diff
@@ -0,0 +1,9 @@
+[run]
+branch = True
+include = C:\projects\clcache\clcache.py
+
+# pytest-cov:
+#   "Note that this plugin controls some options and setting the option in the
+#   config file will have no effect. These include specifying source to be
+#   measured (source option) and all data file handling (data_file and parallel
+#   options)."
```

Comment:
Looks like `include` can be relative and pytest-cov deals with the rest. I'll strip this down accordingly.